src/main/java/com/group2/catanbackend/gamelogic/GameLogicController.java

```diff
@@ -172,21 +229,58 @@ private void computeBuildVillageMoveSetupPhase(BuildVillageMoveDto buildVillageM
 
         if (board.addNewVillage(player, buildVillageMove.getIntersectionID())) {
             player.increaseVictoryPoints(1);
-            board.distributeResourcesSetupPhase(player,buildVillageMove.getIntersectionID());
+            board.distributeResourcesSetupPhase(player, buildVillageMove.getIntersectionID());
             sendCurrentGameStateToPlayers();
         } else
             throw new InvalidGameMoveException(ErrorCode.ERROR_CANT_BUILD_HERE.formatted(buildVillageMove.getClass().getSimpleName()));
 
     }
 
-    private void makeRollDiceMove(RollDiceDto rollDiceDto) {
+    private void makeRollDiceMove(RollDiceDto rollDiceDto, Player player) {
```

Comment:
Gibts einen besonderen grund dass das im dice roll drinnen is? Seems bissl out of place auch wenn es damit zusammenhängt. 
Würde das tbh net ins dice roll dto rein tun